src/components/frontend-engine/yup/types.ts

```diff
@@ -23,6 +23,7 @@ export const YUP_CONDITIONS = [
 	"includes",
 	"excludes",
 	"uinfin",
+	"unitNumberFormat",
```

Comment:
not needed as well, these are meant for common rules, you can declare field-specific rules separately. can refer to `src/components/fields/date-field/types.ts`